### ISSUE TITLE
[Bug][a11y] ETP "X" pop up button needs to have resized the touch target

### ISSUE THREAD

**Issue by Diana-Rus** [mozilla-mobile/fenix#14719]
## Prerequisites:
- Have Google Accessibility Scanner installed.
- Have a clean profile.
## Steps to reproduce
1. Tap on the URL bar and make a search after cnn.com.
2. Wait until ETP is displayed
3. Scan the screen
### Expected behavior
No suggestions are made.
### Actual behavior
Suggestion is to increase the item's size from 24dp x 24dp to 48dp x 48dp or larger
### Device information

* Android device: Google Pixel 3XL (Android9 )
* Fenix version: Nightly 9/4

<details closed><summary>Screen</summary>

![Screenshot_20200904-145054](https://user-images.githubusercontent.com/49859863/92236548-4eb67a00-eebe-11ea-9d20-797bdf28c5ac.png)
</details>

**Comment by sijanr** [comment]
I tried to reproduce the issue in two separate devices, but I wasn't able to.
![no_issues_accessibility_scanner](https://user-images.githubusercontent.com/47872289/93526405-44e43c00-f8ec-11ea-9486-0b7847dc379e.png)

**Comment by Amejia481** [comment]
The issue should be fixed in master!

**Comment by softvision-miralobontiu** [comment]
Verified as fixed on the 1/25 Nightly build with Google Pixel (Anroid 10).
There were no suggestions when scanning the ETP pop-up.

**Comment by Amejia481** [comment]
Awesome! Thanks @softvision-miralobontiu and @fevziomurtekin!

**Comment by fevziomurtekin** [comment]
you're welcome. 😊
